<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="executeButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 114, 69);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Poids</string>
   </property>
  </widget>
  <widget class="QPushButton" name="optionButton">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>440</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 114, 69);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Recommencer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>440</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(128, 114, 69);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(224, 229, 218);
color: rgb(128, 114, 69);
</string>
   </property>
   <property name="text">
    <string>Poids d'une chaine</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>200</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(246, 200, 204);
</string>
   </property>
   <property name="text">
    <string>Donner ch :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>300</y>
     <width>850</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(246, 200, 204);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="chaineInput">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>200</y>
     <width>690</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
gridline-color: rgb(246, 200, 204);
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
